package.json

```diff
@@ -35,6 +35,7 @@
     "onCommand:affinidi.selectActiveProject",
     "onCommand:affinidi.openSchemaBuilder",
     "onCommand:affinidi.initiateIssuanceCsvFlow",
+    "onCommand:affinidi.verifyAVC",
```

Comment:
Please remove "A" from code and filenames, it's a bit inconvenient from developer's perspective

Reply:
Done